frontend/calendar1.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Oklahoma Department of Human Services</title>
+
+        <!--viewport metas go here-->
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="base.css">
+        <link rel="stylesheet" href="styles.css">
+    </head>
+
+    <body>
+        <header class="banner">
+            <img class="logo" src="" alt="DHS logo">
+            <a class="web_title" href="index.html">OKDHS Adult and Family Services Information Center</a>
+
+            <nav class="generalNav">
+                <ul>
+                    <li><a href="profile.html"><img src="" alt="profile">Profile</a></li>
+                    <li><a href="calendar1.html"><img src="" alt="calendar">Calendar</a></li>
+                    <li><a href=""><img src="" alt="upload">Uploads</a></li>
+                    <li><a href="help1.html"><img src="" alt="help">Help</a></li>
+                </ul>
+            </nav>
+        </header>
+
+        <article>
+            <h1>OKDHS AFS Events and Program Deadlines</h1>
+            
+            <section class="dealines">
```

Comment:
This section will be populated and dynamically created using flask and jinga2. This section should also be scroll-able as the table could contain an unknown number of rows.